src/webui/www/private/scripts/client.js

```diff
@@ -1669,7 +1684,15 @@ window.addEventListener("DOMContentLoaded", () => {
 window.addEventListener("load", () => {
     // fetch various data and store it in memory
     window.qBittorrent.Cache.buildInfo.init();
-    window.qBittorrent.Cache.preferences.init();
+    window.qBittorrent.Cache.preferences.init({
+        onSuccess: (pref) => {
+            if (pref.status_bar_external_ip) {
+                const elem = document.getElementById("externalIPs");
+                elem.classList.remove("invisible");
+                elem.previousElementSibling.classList.remove("invisible");
+            }
```

Comment:
Please move it to `processServerState`.

Reply:
That's definitely a better place for this 👍